Tests/TestCommon/MockToken.swift

```diff
@@ -68,19 +68,19 @@ extension Token {
             idToken = try! JWT(JWT.mockIDToken)
         }
         
-        return Token(id: "TokenId",
-                     issuedAt: Date(),
-                     tokenType: "Bearer",
-                     expiresIn: 300,
-                     accessToken: JWT.mockAccessToken,
-                     scope: scopes,
-                     refreshToken: refreshToken,
-                     idToken: idToken,
-                     deviceSecret: deviceSecret,
-                     context: Token.Context(configuration: .init(baseURL: URL(string: "https://example.com/oauth2/default")!,
-                                                                 clientId: "clientid",
-                                                                 scopes: scopes),
-                                            clientSettings: [ "client_id": "clientid" ]))
+        return try! Token(id: "TokenId",
```

Comment:
Do we want to test anything about these tokens once created? Or is the constructor enough of a test by itself?

Reply:
There's the `Tests/AuthFoundationTests/TokenTests.swift` class that exercises the various ways the token can be initialized, including migrating from V1 data.

This mock token utility is simply there to return tokens for use in other tests. 

Force unwrapping makes me feel icky, but as a utility class, if I required all the other tests using this to `try`, it would make the tests messier.